Bottom: f775caac2ce571906b3689c39964b07b5c86f4be
Top:    12e9a1d45ccfac6b159951c9ae690a072dbfd9d1
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-19 20:57:49 +0800

Refresh of drop-vhost_copy_from

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index bfca137..8f6c2f2 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2009,8 +2009,8 @@ int vhost_prefetch_desc_indices(struct vhost_virtqueue *vq,
 	}
 	last_avail_idx = vq->last_avail_idx;
 	vq->avail_idx = vhost16_to_cpu(vq, avail_idx);
-	ret = total = vq->avail_idx - vq->last_avail_idx;
-	total = min(total, num);
+	total = vq->avail_idx - vq->last_avail_idx;
+	ret = total = min(total, num);
 
 	while (total) {
 		int ret2 = vhost_get_avail(vq, indices[0],
